src/integrationTest/resources/TestProject/build.gradle

```diff
@@ -3,16 +3,11 @@ buildscript {
     maven {
       url uri('libs')
     }
-    //flatDir dirs: "C:/Users/i_am_/workspace/umple.gradle/build/libs"
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
   }
   dependencies {
-    // Use the umple jar from the main umple.gradle project
     classpath files('libs/umple-latest.jar')
-    classpath "gradle.plugin.cruise.umple:UmpleGradlePlugin:0.1.2"
-    //classpath "umple.gradle.plugin:UmpleGradlePlugin-0.1.2"
+    // Use the locally built version of the plugin for the tests
+    classpath files('libs/UmpleGradlePlugin-0.1.3.jar')
```

Comment:
@Nava2 This is proving surprisingly difficult. The trick is that the main `umple.gradle` project and `TestProject` are completely separate - nothing passes between them.  That rules out anything other than some kind of `Constants.java` file they could share. But I can't figure out how to work with static variables from another file in `build.gradle`. When I do something like `... = Constants.VERSION_NUMBER` Gradle thinks I'm referring to a `Constants` property that doesn't exist.

I posted a question on SA. Hopefully someone has an idea. I'll keep thinking too.